### PR TITLE
Bug 1994872: Fix manifest path regression

### PR DESCRIPTION
**Pull request by deejross**

### CODE REVIEW

**Review by damemi — COMMENTED** [review]
/approve
/lgtm
looks to be updating https://github.com/openshift/oc/pull/897/files#diff-3de7dd01d29a492d8c331b47c7d7969d013fbf8c1d390e3066771de055d52881R241

**Review by ravisantoshgudimetla — APPROVED** [review]
/lgtm

Thank you for working on this @deejross

**Comment by deejross** [comment]
That test has a known issue that I'm working on between blockers:
```
[sig-arch] events should not repeat pathologically
```

File: pkg/cli/image/imagesource/file_test.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	"runtime"
+	"testing"
+)
+
+func TestGenerateDigestPath(t *testing.T) {
+	path := generateDigestPath("sha256:123456", "mnt", "mirror")
+	desiredPath := filepath.Join("mnt", "mirror", "sha256:123456")
+
+	if runtime.GOOS == "windows" {
+		desiredPath = filepath.Join("mnt", "mirror", "sha256-123456")
```
Comment (soltysh) [inline pkg/cli/image/imagesource/file_test.go:14]: I'd suggest not using `filepath.Join` but rather use a hardcoded string for win and non-win.

**Review by soltysh — COMMENTED** [review]
One nit and this is good to go.

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

**Comment by soltysh** [comment]
/override ci/prow/e2e-agnostic-cmd
we're actively working on it and the failures are not related to this particular PR.

**Pull request merged**